daffodil-runtime1/src/main/resources/test/validation/trace_output.dfdl.xsd

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<xs:schema
+  targetNamespace="http://example.com"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/"
+  xmlns:dfdlx="http://www.ogf.org/dfdl/dfdl-1.0/extensions"
+  xmlns:ex="http://example.com"
+  xmlns="http://example.com"
+  elementFormDefault="unqualified"> 
+
+  <xs:include schemaLocation="org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd"/>
+
+  <xs:annotation> 
+    <xs:appinfo source="http://www.ogf.org/dfdl/" >
+      <dfdl:format ref="ex:GeneralFormat" representation="text" />
+    </xs:appinfo>
+  </xs:annotation>
+	 
+	<xs:element name="output">
+		<xs:complexType>
+			<xs:sequence>
+				<xs:element name="message" type="xs:string" dfdl:inputValueCalc="{dfdlx:trace('hello', 'input')}" />
+			</xs:sequence>
+		</xs:complexType>
+	</xs:element>
+</xs:schema>
```

Comment:
We aren't that strict about this, but we usually prefer files end with a newline. There should be a setting in your IDE to make sure files are saved with a newline.